packages/strapi-plugin-upload/admin/src/components/DropdownSection/index.js

```diff
@@ -7,7 +7,7 @@ const DropdownSection = styled.div`
   position: absolute;
   top: 38px;
   left: 0;
-  z-index: 1;
+  z-index: 1051;
```

Comment:
The fixed header has a `z-index: 1050`, so can you set it to a lower value to avoid the dropdown section to be above on the listview scroll?

Reply:
I set the z-index to 1051 to avoid the Selected item border to be above the dropdown. But I just see that you fixed this bug.